app/src/matrix_transform.c

```diff
@@ -13,14 +13,15 @@
 
 #define _TRANSFORM_ENTRY(i, _)                                                                     \
```

Comment:
Not an issue with this PR, but since you're editing this code already, this would be nice to fix. All identifiers that begin with an underscore followed by a capital letter are reserved for use by the C compiler.

```suggestion
#define TRANSFORM_ENTRY(i, _)                                                                      \
```

Reply:
DONE